cmd/eventprocessor.go

```diff
@@ -123,27 +142,45 @@ func main() {
 		metricsSink.IncrCounter(messagesReceived, 1)
 
 		message := *messages[0]
-		log.Debugf("Message: %v", message)
 
+		loggingFields := log.Fields{"msg id": *message.MessageId}
+
+		log.WithFields(loggingFields).Infof("Extracting SNS message from %s", *message.MessageId)
+
+		snsMessageOK := true
 		sns, err := SNSMessageFromRawMessage(*message.Body)
 		if err != nil {
-			warnErrorf(err.Error())
-			errorDelay()
-			continue
+			snsMessageOK = false
```

Comment:
Unless there are special circumstances that would dictate other, it would make sense to do that as well.